spanner/api/Spanner.Samples/Performance.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_data_with_timestamp_column]
+
+public class Performance
+{
+    public int SingerId { get; set; }
+    public int VenueId { get; set; }
+    public System.DateTime EventDate { get; set; }
```

Comment:
```suggestion
    public DateTime EventDate { get; set; }
```
And add using System;

Reply:
We have not added any import in classes that has been used in more than one sample as it might be conflict on import.

Reply:
Ahh, that's a good point. And another reason to move these to the sample files where they are being used.